src/curated-corpus/pages/ProspectingPage/ProspectingPage.tsx

```diff
@@ -1,9 +1,8 @@
 import React, { useEffect, useState } from 'react';
 import { DateTime } from 'luxon';
-import { Box, Button, Grid, Hidden } from '@material-ui/core';
-import RefreshIcon from '@material-ui/icons/Refresh';
-import FilterListIcon from '@material-ui/icons/FilterList';
-import AddIcon from '@material-ui/icons/Add';
+import { Box, Button, Grid, Hidden } from '@mui/material';
+import { Refresh, FilterList } from '@mui/icons-material';
+import AddIcon from '@mui/icons-material/Add';
```

Comment:
I'm curious why you used a shorter import here for icons. I'm excited they can be imported from just `@mui/icons-material` without the `Icon` at the end! Yay! But I also like how the code is more readable when it's clear the thing inserted in the markup is an icon.

Reply:
I think the key here is consistency. This is the one and only place I'm seeing these shorter imports for icons. We should do this everywhere or use `XyzIcon` imports everywhere, not both.

Reply:
Yep I agree. I started with this page and wanted to do a condensed import but turns out not all icons can be imported from the base `/icons-material` so in the other components went with separate imports. Fixed that imports here!